### PR TITLE
Add fctiw, fctiwz, frsp, fabs, fnabs, fmr, and fneg to the JIT.

### PR DESCRIPTION
**Pull request by MoochMcGee**
This was done with implementations ported from Dolphin.

### CODE REVIEW

**Comment by brett19** [comment]
I would like to run these against hardware before we merge.  I'll get @exjam to look at that.  Otherwise they look perfect :)

**Comment by exjam** [comment]
Real men merge now, test later.

File: src/cpu/jit/jit_float.cpp [hunk 1]
```diff
@@ -14,6 +18,74 @@ namespace jit
...
+	   a.movdqu(a.xmm0, a.ppcfpr[instr.frB]);
+	   a.movsd(a.ppcgpr[instr.frD], a.xmm0);
+
+	   return true;
+   }
+
+   static bool fsign(PPCEmuAssembler& a, Instruction instr)
+   {
+	   a.movdqu(a.xmm0, a.ppcfpr[instr.frB]);
+	   a.xorps(a.xmm0, a.ptr(psSignBits);
```
Comment (Tilka) [inline src/cpu/jit/jit_float.cpp]: This doesn't even compile.

Comment (exjam) [inline src/cpu/jit/jit_float.cpp]: Yes I will setup some CI service tomorrow haha.

**Pull request merged**